doc/classes/EditorSceneImporter.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="EditorSceneImporter" inherits="Reference" category="Core" version="3.1">
 	<brief_description>
+		Used to import scenes from 3D modeling files, e.g. [code].dae[/code], [code].obj[/code], [code].glb[/code]...
```

Comment:
Avoid `...` and passive voice: https://docs.godotengine.org/en/stable/community/contributing/docs_writing_guidelines.html

Is it the class responsible for importing any kind of 3d asset, including FBX? If so the main formats should be fbx and gltf (= glb). I'll let someone else confirm this but I think they're going to be the most recommended formats from Godot 3.2, with the escn files exported from Blender, over collada and obj

`Imports scenes from 3d files, e.g. `

Reply:
EditorSceneImporter is the base class for importers of scenes. It is inherited by EditorSceneImporterCollada (.dae), EditorOBJImporter (.obj), EditorSceneImporterGLTF (.gltf), and EditorSceneImporterAssimp (.fbx and stuff).
It can also be inherited from a script, for custom importers.

Reply:
So I think is better to avoid mentioning the file formats then?